rts/Lua/LuaSyncedRead.cpp

```diff
@@ -6236,20 +6238,18 @@ int LuaSyncedRead::GetUnitCurrentCommand(lua_State* L)
  * Same as `Spring.GetCommandQueue`
  *
  * @param unitID integer
- * @param count integer Number of commands to return, `-1` returns all commands, `0` returns command count.
+ * @param count integer Maximum amount of commands to return, `-1` returns all commands.
  * @return Command[] commands
  */
 /***
  * Get the count of commands for a unit.
  *
+ * @deprecated
```

Comment:
Add the same deprecation message to this line pls. e.g.:

```suggestion
 * @deprecated This overload is deprecated, prefer to use `Spring.GetUnitCommandCount(unitId)`.
```

Otherwise the deprecation message is only visible at runtime.

Reply:
Oh, I looked at some lua-ls docs specifically looking for this and didn't find that I could pass a message, thank you.

Reply:
Yeah, it's not documented there, but I'm pretty sure it works just fine. Perhaps just because it shows the literal docs when you hover the error. In any case I believe emmylua handles it.